package.json

```diff
@@ -41,11 +38,12 @@
     "mock-require": "^3.0.3",
     "nock": "^13.0.11",
     "nyc": "^15.1.0",
+    "oclif": "^3.0.1",
     "semantic-release": "^17.2.2",
     "stdout-stderr": "^0.1.13"
   },
   "engines": {
-    "node": ">=10.0.0"
+    "node": "^14.18 || ^16.13 || >=18"
```

Comment:
again i believe removing nodejs12 will cause some issues in our internal CI since we run it in node12: https://git.corp.adobe.com/nui/cli/blob/master/.circleci/config.yml#L88

Reply:
Yes, please keep node:12 for now.

Reply:
Why not just stop running your tests on node:12 since it is end of life and start running tests on 14/16/18?
https://nodejs.org/en/about/releases/